agent/acs/model/api/api-2.json

```diff
@@ -228,7 +228,7 @@
         "dependsOn":{"shape":"ContainerDependencies"},
         "startTimeout":{"shape":"Integer"},
         "stopTimeout":{"shape":"Integer"},
-        "logRouter":{"shape":"LogRouter"}
+        "firelensConfiguration":{"shape":"FirelensConfiguration"}
```

Comment:
Do we not need to update the state file version for this? don't think I see that change in this cr

Reply:
there are some changes in agent/statemanager/state_manager.go

Reply:
oh yep i see it now 👍